Akinator.Api.Net/Utils/AkiUrlBuilder.cs

```diff
@@ -6,16 +6,25 @@ namespace Akinator.Api.Net.Utils
 {
     internal static class AkiUrlBuilder
     {
-        public static string NewGame(ApiKey apiKey, Language language, ServerType serverType)
+        public static string NewGame(ApiKey apiKey, Language language, ServerType serverType, bool child_mode)
         {
+            var child_switch = String.Empty;
+            var question_filter = String.Empty;
+
             var server = ServerSelector.GetServerFor(language, serverType);
             if (string.IsNullOrEmpty(server))
             {
                 throw new InvalidOperationException($"No server does match the language {language} and server type {serverType}.");
             }
 
+            if (child_mode == true)
+            {
+                child_switch = "true";

```

Comment:
can you do it like
```
if (childMode)
{
   return $"WITH BOTH NEW PARAMETERS"
}

return $"THE OLD URL"
```

instead?

Reply:
Okay, but note that Akinator web app adds these parameters with no value with child mode disabled

Reply:
Okay, I understand. Then feel free to keep it like you already had before.